mypy_django_plugin/transformers/functional.py

```diff
@@ -20,6 +23,12 @@ def resolve_str_promise_attribute(ctx: AttributeContext) -> MypyType:
     str_info = helpers.lookup_fully_qualified_typeinfo(helpers.get_typechecker_api(ctx), "builtins.str")
     assert str_info is not None
     str_type = Instance(str_info, [])
+
+    # TODO: [mypy 1.16+] Remove this workaround for passing `msg` to `analyze_member_access()`.
+    extra: dict[str, Any] = {}
+    if "msg" in inspect.signature(analyze_member_access).parameters:  # mypy < 1.16
```

Comment:
Bummer. Looks like `inspect.signature()` won't work because of mypyc...

```
ValueError: no signature found for builtin <built-in function analyze_member_access>
```